checks/apps/gromacs/gromacs_benchpep-h.py

```diff
@@ -0,0 +1,174 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
+@rfm.simple_test
+class lumi_gromacs_pep_h(gromacs_check):
+    benchmark_info = parameter([
+        ('benchPEP-h', 1.27e-04, 1.6e-05), 
```

Comment:
I'll double-check these values, what is the tolerance used in comparing with these reference values?